README.md

```diff
@@ -38,11 +52,11 @@ _Note: If coming from a deb install, the directory structure is different and yo
 | Latest Release | [FlatHub Page](https://flathub.org/apps/details/io.lbry.lbry-app)         | [AUR Package](https://aur.archlinux.org/packages/lbry-app-bin/)                           | [Pi Installer](https://lbrypi.com)          |
 | Maintainers    | [@choofee](https://github.com/choffee)/[@iuyte](https://github.com/iuyte) | [@kcseb](https://github.com/kcseb)/[@TimurKiyivinski](https://github.com/TimurKiyivinski) | [@Madiator2011](https://github.com/kodxana) |
 
-## Usage
+<h2 align="left">Usage</h2>
```

Comment:
We shouldn't need to change these, the links will still work.

Reply:
I have changed the `h2` to `##`.